src/d3a/models/strategy/external_strategies/load.py

```diff
@@ -284,7 +284,9 @@ def _bid_aggregator(self, arguments):
                 arguments["price"],
                 arguments["energy"],
                 replace_existing=replace_existing,
-                **arguments)
+                attributes=arguments.get("attributes", None),
+                requirements=arguments.get("requirements", None)
```

Comment:
```suggestion
                attributes=arguments.get("attributes"),
                requirements=arguments.get("requirements")
```

Reply:
Indeed, will fix it in all occurrences, thanks!